### ISSUE TITLE
Add morphology model illustrations to the docs

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#242]
We should add an image (and maybe radial profile) to the docstring of each morphology model like e.g. [Shell2D](https://gammapy.readthedocs.org/en/latest/api/gammapy.morphology.Shell2D.html).

Also a "morphology model gallery" in the [gammapy.morphology](https://gammapy.readthedocs.org/en/latest/morphology/index.html) docs or an interactive notebook where the user can adjust the model parameters (and maybe Gauss PSF convolution) would be great.

Basically it should be easy for users to see which models are available and understand their parameters.

**Comment by cdeil** [comment]
Anyone interested to work on spatial model docs?

We need something that gives an overview of available models and links to each class (could be with or without plots):

- https://astropy-regions.readthedocs.io/en/latest/shapes.html
- http://cta.irap.omp.eu/ctools/users/user_manual/models.html
- https://fermi.gsfc.nasa.gov/ssc/data/analysis/scitools/source_models.html

See related https://github.com/gammapy/gammapy/issues/241#issuecomment-535971753

**Comment by adonath** [comment]
Again I vote for a "model gallery", a single docs page where all models are listed and illustrated.

**Comment by cdeil** [comment]
The task here is to improve the spatial model section here: https://github.com/gammapy/gammapy/blob/master/tutorials/models.ipynb

Make one or several nice Figures that illustrate all spatial models that we have:
https://github.com/gammapy/gammapy/blob/ee2535af97986f8df550b1fd9c0f2c0ae2d3c877/gammapy/modeling/models/__init__.py#L21-L29

Anyone interested / available to do this in October?

**Comment by adonath** [comment]
Implemented in #2737.
